ivre/tools/ipdata.py

```diff
@@ -92,21 +94,23 @@ def my_parse_args():
                 ivre.db.db.data.ensure_indexes()
     if args.download:
         ivre.geoiputils.download_all(verbose=not args.quiet)
+    dbtofeed = {}
+    if not args.dont_feed_ipdata_cols:
+        dbtofeed["feedipdata"] = [ ivre.db.db.passive ]
+        if args.update_nmap_db:
```

Comment:
I assumed the purpose of the "dont-feed-ipdata-cols" option was "Update my GeoIP data, but do not update what has already been scanned, whether it's active or passive".

Maybe we could reverse the problem by removing the "dont-feed-ipdata-cols" option, replacing it by both an "update-passive-db" option and the "update-nmap-db" option.

This would make sense to explicitely specify whether to update one db or another, or both.

Would this solution do the trick?

I'll try to be PEP8-compliant from now on.